### ISSUE TITLE
[pni] Product Page Accessibility Testing

### ISSUE THREAD

**Issue by youriwims** [mozilla/foundation.mozilla.org#4006]
Morphed to epic to track all the issues identified in comment #2

Related Issue: #3934

**Comment by Pomax** [comment]
# Issues identified

## Serious

### Join us content is invisible instead of display:none

This means tabbing to the main content will "get stuck" cycling through everything in the "join us" newsletter slide-out panel, without any indication that that's where focus is.

We can either make that slide-out be display:none with extra work to make sure the transitions work properly, or we can make it so that if there is focus anywhere inside the slide-out, the slide-out itself gets revealed.

### PNI nav contrast

Text color is too low contrast (currently #999999, which has contrast 2.85, and needs to be #707070 or lower to pass)

![image](https://user-images.githubusercontent.com/177243/81863049-76ed2800-951f-11ea-81c2-446861c9b12b.png)

### creepometer hint contrast

Text color is too low contrast (currently #999999, which has contrast 2.85, and needs to be #707070 or lower to pass)

![image](https://user-images.githubusercontent.com/177243/81863107-879d9e00-951f-11ea-8372-811fac1d0ad7.png)

## Moderate

### `<figcaption>` cannot be nested

The `<figcaption>` that we use in `<figure>` elements for each product cannot be nested inside another element and needs to be a direct child of `<figure>`

![image](https://user-images.githubusercontent.com/177243/81863466-20341e00-9520-11ea-82f9-304afd800e61.png)

### Headings "skip" from h1 to h3

Headings should not skip levels, so this h3 should be an h2, or not use heading markup.

![image](https://user-images.githubusercontent.com/177243/81863675-71dca880-9520-11ea-90bd-f2e991fde5b2.png)

### Missing `<main>` element

This element is relied on to navigate accessibility tools directly to the start of the main content, we don't have one.

###

**Comment by Pomax** [comment]
@kristinashu @mmmavis there's a few ways we can move forward on this one, some I can just fix by replacing what we have with best-practice markup, but some might require a design eye to decide what we want to do, such as the text contrast and newsletter slide-down dialog.

**Comment by Pomax** [comment]
With all this work landed, a new a11y pass needs to be done to determine what work is still left on the PNI a11y testing front.

**Comment by Pomax** [comment]
## homepage

- [ ] Text for `The 😮 below shows how creepy users find these products. ` should be `gray-60`
- [ ] Footer text "Protect the internet as a global public resource" should be `<p`> with h5-heading CSS, no an actual `<h5>`
- [ ] Footer text "More about us" should be a `<p>` as above

**Comment by Pomax** [comment]
## Category page

- [ ] company + name should be `<p>`, not `<h3>`

**Comment by Pomax** [comment]
## About page

- [ ] side nav should use `gray-60`
- [ ] inner `<main>` should be `<div>`

**Comment by Pomax** [comment]
And we're done.
